### PR TITLE
Feat: Add C4GT 2022 Page

### LINKED ISSUE #6: UI for the C4GT 2022 page
## Description
We need to create the C4GT'22 page for the website [c4gt-2022-projects](https://www.codeforgovtech.in/copy-of-c4gt-2022) and implement Cypress tests to ensure its functionality. This task involves designing and developing the responsive C4GT'22 page layout and adding relevant content. The Cypress tests will ensure that the page functions as expected and all components are working correctly.

## Goals
 - [ ] Design and develop the C4GT'22 page layout
 - [ ] Add relevant content to the C4GT'22 page
 - [ ] Make it responsive
 - [ ] Implement Cypress tests for the C4GT'22 page

## Expected Outcome
The C4GT'22 page should have an attractive and user-friendly design, showcasing the organization's mission and providing easy navigation. The content should be engaging and informative. The Cypress tests should cover the functionality of the C4GT'22 page, ensuring that all interactive elements and features are working properly.

## Implementation Details
Next.js is used as the framework for the website. Cypress will be the chosen testing framework for the Cypress tests.

## Mockups / Wireframes
[Link](https://www.codeforgovtech.in/copy-of-c4gt-2022)
Screenshot from 2023-06-22 11-48-25

## Project
Code for GovTech

## Tech Skills Needed:
JavaScript, Next.js, Cypress

## Mentor(s)
@Shruti3004 

## Complexity
Medium

## Category
UI/UX Enhancements

### PR DESCRIPTION
**Pull request by void-ness**
Fixes #6 
Added a page for the C4GT 2022 program with the relevant content.

## Goals
- [x] Design and develop the C4GT'22 page layout
- [x] Add relevant content to the C4GT'22 page
- [x] Make it responsive
- [x] Implement Cypress tests for the C4GT'22 page

## Screenshots
1. image 
2. image

Responsive | Designs
--- | --- 
image | image

### CODE REVIEW

**Comment by void-ness** [comment]
Hey @Shruti3004 , can you please have a look at the above PR once. I have done the suggested changes as per the open issue and would love your feedback on the same.
Thanks

**Comment by Shruti3004** [comment]
> Hey @Shruti3004 , can you please have a look at the above PR once. I have done the suggested changes as per the open issue and would love your feedback on the same. Thanks

Hey @void-ness yes sure.

File: cypress/e2e/c4gt22/test1.cy.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+describe("C4GT 2022 Page components render test", () => {
```
Comment (Shruti3004) [inline cypress/e2e/c4gt22/test1.cy.js:1]: Please fix the file name should be `index.cy.js`

File: cypress/e2e/c4gt22/test1.cy.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+      Cypress.dom.isVisible($el);
+    });
+  });
+
+  it("should display the header section correctly", () => {
+    cy.get("h2").should("contain", "Highlights from 2022");
+    cy.get("p").should("contain", "Code for GovTech (C4GT) was launched in May 2022, as an ecosystem-led");
+    cy.get("span").should("contain", "Here are some of the key highlights from 2022.");
+  });
+});
```
Comment (Shruti3004) [inline cypress/e2e/c4gt22/test1.cy.js:17]: PLease create `assets`` folder in the public and then inside assets create `contributors` and  `mentors`

**Comment by Shruti3004** [comment]
Hey @void-ness can you please take a pull from the main branch there we are using `Mulish-Regular` font can we use it?

File: cypress/e2e/c4gt22/test1.cy.js (hunk 1 above)
Comment (void-ness) [reply cypress/e2e/c4gt22/test1.cy.js]: Fixed in b42fa5f

File: cypress/e2e/c4gt22/test1.cy.js (hunk 2 above)
Comment (void-ness) [reply cypress/e2e/c4gt22/test1.cy.js]: Fixed in 351cdf1

**Comment by Shruti3004** [comment]
LGTM @void-ness

**Review by Shruti3004 — APPROVED** [review]

**Pull request merged**